shapeshift/web
# feat: allow degen thor swaps

## Description

Fundus may be safu, may not be. You've been warned, your call.

## Pull Request Type


## Issue (if applicable)



closes https://github.com/shapeshift/web/issues/5804

## Risk



- Ensure the warning does not appear in case the active quote meets the minimum

## Testing



- Try and get THOR quotes for lower amounts
- Notice you now get an actual quote instead of an error, are warned that this amount is unsafe, and have to click "I understand" before being able to continue. The minimum is also displayed in the warning in case you wish to continue with a safe amount instead
- After clicking "I understand", you're able to continue (or are met with the usual error-handling e.g insufficient funds)
- Ensure inputting the minimum recommended amount lets you proceed without a warning

### Engineering



- ☝🏽 

### Operations



- ☝🏽 

## Screenshots (if applicable)


https://github.com/shapeshift/web/assets/17035424/9a7db511-54ee-4ae2-9374-e73a0df9d3bb

<img width="608" alt="Screenshot 2023-12-08 at 00 01 51" src="https://github.com/shapeshift/web/assets/17035424/5b8ddbba-e7d3-48f8-b3df-f244b91fbdbf">
<img width="587" alt="Screenshot 2023-12-08 at 00 02 05" src="https://github.com/shapeshift/web/assets/17035424/8926c16c-e041-4a31-b5e4-dbf72a01e926">
